src/Tribe/Settings.php

```diff
@@ -5,822 +5,911 @@
 	die( '-1' );
 }
 
-use Tribe\Admin\Pages as AdminPages;
+use Tribe\Admin\Pages as Admin_Pages;
 
-if ( ! class_exists( 'Tribe__Settings' ) ) {
+if ( did_action( 'tec_settings_init' ) ) {
+	// If We've already initialized, don't do it again - return the current instance.
+	return tribe( 'settings' );
```

Comment:
Why the `return tribe( 'settings' );` here, instead of just `return;`?

Reply:
Actually a fair question - I was thinking at the time I needed to return the class, but I don't here. Need more coffee?